### PR TITLE
SONAR-6486 WS permissions/users project permissions

### PR DESCRIPTION
**Pull request by teryk**
check the last commit only: https://github.com/SonarSource/sonarqube/commit/19b21f63378d129b177d673118ebcbf429d3a8d4

### CODE REVIEW

File: server/sonar-server/src/main/java/org/sonar/server/permission/ws/UsersAction.java [hunk 1]
```diff
@@ -57,46 +71,53 @@ public void define(WebService.NewController context) {
...
-      permissionQuery.search(query);
-    }
+    PermissionQuery permissionQuery = buildPermissionQuery(wsRequest, project);
+    Permissions.UsersResponse usersResponse = usersResponse(permissionQuery, page, pageSize);
 
-    UserWithPermissionQueryResult usersResult = permissionFinder.findUsersWithPermission(permissionQuery.build());
+    writeResponse(wsResponse, usersResponse);
+  }
+
+  private void writeResponse(Response wsResponse, Permissions.UsersResponse usersResponse) {
```
Comment (SonarTech) [inline server/sonar-server/src/main/java/org/sonar/server/permission/ws/UsersAction.java]: MAJOR Make "writeResponse" a "static" method. [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS2325)

**Comment by SonarTech** [comment]
SonarQube analysis reported 3 issues:
* MAJOR 3 major

Watch the comments in this conversation to review them.
Note: the following issues could not be reported as comments because they are located on lines that are not displayed in this pull request:
* MAJOR Add a 'package-info.java' file to document the 'src/main/java/org/sonar/server/permission' package (org.codehaus.sonar:sonar-server:src/main/java/org/sonar/server/permission) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1228)
* MAJOR Add a 'package-info.java' file to document the 'src/main/java/org/sonar/server/permission/ws' package (org.codehaus.sonar:sonar-server:src/main/java/org/sonar/server/permission/ws) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1228)

**Comment by SonarTech** [comment]
SonarQube analysis reported 2 issues:
* MAJOR 2 major

Watch the comments in this conversation to review them.
Note: the following issues could not be reported as comments because they are located on lines that are not displayed in this pull request:
* MAJOR Add a 'package-info.java' file to document the 'src/main/java/org/sonar/server/permission' package (org.codehaus.sonar:sonar-server:src/main/java/org/sonar/server/permission) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1228)
* MAJOR Add a 'package-info.java' file to document the 'src/main/java/org/sonar/server/permission/ws' package (org.codehaus.sonar:sonar-server:src/main/java/org/sonar/server/permission/ws) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1228)

**Comment by sns-seb** [comment]
@teryk  back to you

**Pull request closed without merging**